CUSTOMISING.md

```diff
@@ -110,6 +110,32 @@ damap:
       password: your-datasource-password
       db-kind: postgresql # your database type
 ```
+### Custom Database Changes
+
+If you want to customize [liquibase](https://www.liquibase.org/get-started) for your own institutional backend:
+- Create a new folder in db with your institutions name - like e.g. tuwien
+- Create your own root changelog and change the liquibase path in application.yaml to point to your new root changelog
+```yaml
+liquibase:
+  migrate-at-start: true
+  change-log: db/tuwien/newChangeLog-root.yaml
+```
+- Your new root changelog should look something like this:
+```yaml
+databaseChangeLog:
+  - include:
+      file: db/damap/changeLog-root.yaml
```

Comment:
this should be updated to the new address of the file
at/ac/tuwien/damap/db/changeLog-root.yaml